test/Microsoft.AspNet.Mvc.TagHelpers.Test/Internal/FileVersionProviderTest.cs

```diff
@@ -38,54 +37,188 @@ public void AddsVersionToFiles_WhenCacheIsAbsent(string filePath, string expecte
             Assert.Equal(expected, result);
         }
 
+        [Fact]
+        public void AddFileVersionToPath_CachesNotFoundResults()
+        {
+            // Arrange
+            var path = "/wwwroot/file.txt";
+            var fileProvider = GetMockFileProvider(
+                path,
+                pathStartsWithAppName: false,
+                fileDoesNotExist: true);
+            var fileVersionProvider = new FileVersionProvider(
+                fileProvider,
+                new MemoryCache(new MemoryCacheOptions()),
+                GetRequestPathBase());
+            var mockFileProvider = Mock.Get(fileProvider);
+
+            // Act 1
+            var result = fileVersionProvider.AddFileVersionToPath(path);
```

Comment:
Isn't it enough just to call this twice and do the `Times.Once()` asserts once? Do we need two acts and asserts?